backend/typescript/middlewares/validators/campValidators.ts

```diff
@@ -168,29 +165,34 @@ export const updateCampDtoValidator = async (
   res: Response,
   next: NextFunction,
 ) => {
-  const body = JSON.parse(req.body.data);
+  let body;
+  try {
+    body = JSON.parse(req.body.data);
+  } catch (e: unknown) {
+    return res.status(400).send(getErrorMessage(e));
+  }
+  if (!body) {
+    return res.status(400).send("JSON parsing failed");
+  }
   if (!validatePrimitive(body.name, "string")) {
     return res.status(400).send(getApiValidationError("name", "string"));
   }
-  if (body.description && !validatePrimitive(body.description, "string")) {
+  if (!validatePrimitive(body.description, "string")) {
```

Comment:
redundant, the `validatePrimitive` method checks against `null` and `undefined`?